config/fam_client_interface_config.yaml

```diff
@@ -0,0 +1,18 @@
+#FAM Client Interface Server Config file
+
+# List of memory servers that is associated with metadata manager.
+#memsrv_ip :
+#- 0:<ip_address_1>
+#- 1:<ip_address_2>
+#- 3:<ip_address_3>
+#- 2:<ip_address_4>
+memsrv_ip :
+- 0:127.0.0.1
```

Comment:
we need to add port for both RPC's. 
We can make format as 
 - memserver_id:memserver_ip:port

Same comment below